impl/client/src/main/javascript/web/pentaho/visual/role/mappingAttribute.js

```diff
@@ -144,15 +143,13 @@ define([
          * the containing mapping's [levelEffective]{@link pentaho.visual.role.Mapping#levelEffective}
          * is [quantitative]{@link pentaho.visual.role.Level.Type#isQuantitative}.
          *
-         * This key is composed by the value of the [name]{@link pentaho.visual.role.MappingAttribute#name} and
-         * the [aggregation]{@link pentaho.visual.role.MappingAttribute#aggregation} properties.
+         * This key is composed by the value of the [name]{@link pentaho.visual.role.MappingAttribute#name} property.
          *
          * @type {string}
          * @readOnly
          */
         get keyQuantitative() {
```

Comment:
@davidmsantos90 The reason for the existence of `keyQualitative` and `keyQuantitative` in the first place was for the difference in uniqueness caused by the `aggregation` property.
Once this property has been removed, there is no need for two key properties. Maybe even there's no need to even create a getter for this and the user should just know that the key is the `name` property.

We cannot yet use `Complex#$key`, and override it to return the value of `name`, because `name` is not immutable and the list implementation (used by `Mapping#attributes`) is not prepared for keys to change along the way. So, we cannot improve things on this end, for now, and thus need to continue explicitly validating attribute `name` uniqueness in the attributes list...